package.json

```diff
@@ -22,31 +22,28 @@
   },
   "devDependencies": {
     "aws-sdk": "~2.2.43",
-    "babel-plugin-feature-flags": "~0.2.0",
+    "babel-plugin-feature-flags": "0.2.0",
```

Comment:
Correct, we only push a `release-2-x` branch when it is no longer the current stable version.  `release` is the right target for this change.  Though I suspect that we should probably also fix `lts-2-4`...